src/help.js

```diff
@@ -0,0 +1,29 @@
+import shell from 'shelljs';
+
+const badCommands = ['ShellString', 'cd', 'pushd', 'popd', 'dirs'];
```

Comment:
There is a `CMD_BLACKLIST` already defined in `shx.js`.  Suggest pulling that and the `ERROR_CODES` into a `config.js` or something so it can imported into both places.

Then you can filter based on the same list.